bika/lims/content/bikasetup.py

```diff
@@ -757,6 +757,45 @@ class PrefixesField(RecordsField):
                           "will be displayed too. By default, disabled.")
         )
     ),
+    BooleanField(
+        'SamplingBarEnabledAnalysisRequests',
+        schemata="Analyses",
+        default=False,
+        widget=BooleanWidget(
+            label=_("Enable the sampling bar in Analysis Requests lists"),
+            description=_(
+                "If you enable this field, the Analysis Requests Lists will"
```

Comment:
Typo: "If enabled, the user will be able to filter lists of Analysis Requests by several criteria, e.g.:  ......"